docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -46,3 +46,28 @@ SELECT mapSubtract(([toUInt8(1), 2], [toInt32(1), 1]), ([toUInt8(1), 2], [toInt3
 │ ([1,2],[-1,0]) │ Tuple(Array(UInt8), Array(Int64)) │
 └────────────────┴───────────────────────────────────┘
 ````
+
+## mapFill {#function-mapfill}
+
+Syntax: `mapFill(key, value [, max])`
+
+Finds minimum and maximum in the `key` array, and fills the `key` array with keys which are missing between the minimum and maximum,
+and correspondingly fills `value` array with default values for added keys.
+
+If `max` is specified, `mapFill` uses it as the maximum.
+
+`key` and `value` are arrays of integer types.
+
+The number of elements in `key` and `value` must be the same for each row.
+
+Returns a tuple of two arrays: keys in sorted order, and values the corresponding keys.
```

Comment:
It might also be worth mentioning that only the first `value` (in order of appearing) gets associated with the `key`

Reply:
Done.